### PR TITLE
Refactor router just a little

### PR DESCRIPTION
**Pull request by oskarcieslik**

### CODE REVIEW

File: src/router.js [hunk 1]
```diff
@@ -12,37 +12,41 @@ export class Router {
...
-    this.register(...['PUT', ...args]);
+    this.request('PUT', ...args);
   }
 
   delete(...args) {
-    this.register(...['DELETE', ...args]);
+    this.request('DELETE', ...args);
+  }
+
+  request(method, ...args) {
```
Comment (zzarcon) [inline src/router.js]: Not sure about this name, `request` looks more like doing the actual request to me... What do you think?

Comment (oskarcieslik) [inline src/router.js]: Good point, maybe registerRequest?

Comment (oskarcieslik) [inline src/router.js]: As we discussed, it's going to be merged with register()

**Comment by zzarcon** [comment]
Ready 🚀

**Pull request merged**